diff --git a/bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py b/bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py
index e751fcf3..92fed2c4 100644
--- a/bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py
+++ b/bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py
@@ -26,6 +26,7 @@ import os
 import glob
 import datetime
 import tempfile
+import sys
 from pathlib import Path
 
 from PyQt5 import QtWidgets
@@ -41,11 +42,12 @@ from qgis.core import (
 from qgis.utils import iface
 from qgis.gui import QgsMapLayerComboBox, QgsFieldComboBox
 
+sys.path.insert(0, os.path.dirname(__file__))
 from smoderp2d.runners.qgis import QGISRunner
 from smoderp2d.core.general import Globals, GridGlobals
 from smoderp2d.providers import Logger
 from smoderp2d.exceptions import ProviderError, ComputationAborted
-from bin.base import arguments, sections
+from base import arguments, sections
 
 from .connect_grass import find_grass_bin
 from .custom_widgets import HistoryWidget
